server/routes/utilMongoose.js

```diff
@@ -782,6 +827,17 @@ router.get('/data/clear/collegeRep', async (req, res) => {
     }
 })
 
+router.post('/addEvent', async (req, res) => {
+    try {
+        await createEvent(req.body.creator, req.body.title, req.body.date, req.body.link, req.body.description, req.body.school, req.body.startYear, req.body.endYear)
```

Comment:
We're still having the creator be included in the API request? I thought we were gonna pull that out from the request info?

Reply:
This is for utilMongoose which as far as I know does not include passport auth stuff because it is only really used for seeding/testing. Or maybe I am not fully understanding your question.

Reply:
Sorry, yeah I might've asked this in the wrong place.. I think what I was referring to was the in the place we submit the request, does the creator (or creator id) need to be specified in the body of the request? Or can that be inferred via the passport auth stuff?

Reply:
Oh that makes sense! Yes, the end points were updated so the user information is retrieved from the request without having the FE send it back.